packages/grant-explorer/src/features/discovery/ExploreRoundsPage.tsx

```diff
@@ -18,14 +18,17 @@ const ExploreRoundsPage = () => {
   // Pass the filter from the search params and build the graphql query
   const rounds = useFilterRounds(filter, getEnabledChains());
 
+  const publicRounds = rounds.data?.filter(round => round.roundMetadata.roundType?.toLowerCase() !== "private");
+  rounds.data = publicRounds;
```

Comment:
this will surely blow up in the future in unexpected ways.

Reply:
Yup! We likely have make this as custom function on the indexer. Will make an issue to track and fix it up this week

Reply:
https://github.com/gitcoinco/grants-stack/issues/3433